fbchat/client.py

```diff
@@ -325,8 +326,8 @@ def isLoggedIn(self):
         :rtype: bool
         """
         # Send a request to the login url, to see if we're directed to the home page
-        r = self._cleanGet(self.req_url.LOGIN)
-        return 'home' in r.url
+        r = self._cleanGet(self.req_url.LOGIN, allow_redirects=False)
+        return 'home' in r.headers['Location']
```

Comment:
This fails if the `Location` header isn't set, as when the client isn't logged in, and therefore isn't redirected. I went ahead and fixed it with by testing if `'Location' in r.headers`